win32/ioutil.h

```diff
@@ -795,6 +795,7 @@ __forceinline static ssize_t php_win32_ioutil_readlink(const char *path, char *b
 	}
 	memcpy(buf, ret_buf, ret_buf_len + 1);
 
+	free(ret_buf);
```

Comment:
What about the error case above? It can be hit if ret_buf is non-null but too large.

Reply:
Good catch, thanks! (I already pushed a respective commit to the branch, but it doesn't show up here, yet.)